test/system/test_cvp_client.py

```diff
@@ -327,7 +326,7 @@ def test_get_except_fail_reconnect(self):
         try:
             # Try a get request and expect a CvpSessionLogOutError
             result = self.clnt.get('/cvpInfo/getCvpInfo.do')
-        except (CvpSessionLogOutError, CvpApiError) as error:
+        except (CvpSessionLogOutError, CvpApiError):
```

Comment:
Same as above for Py2 vs Py3. I am fine with updates that make code more Py3 standard and deprecate Py2 standards

Reply:
Okay. This except block will work for both py2 and py3. We are not using the 'error ' variable in the below code, so we removed the unused variable from except block.